src/content/events/2023-08-17_ARC-Club_HHU.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "ARC Club HHU"
+excerpt: "a two-day adventure to prepare your lab for the ARC universe"
+category: Training
+mode: On-site
+audience: [Users]
+when:
+    start: 2023-08-17T09:00:00
+    end: 2023-08-18T17:00:00
+location:
+  short: HHU
+  address: HHU, ZSL 22.07, Room U1.21
+  url: https://www.ceplas.eu/en/contact/how-to-get-there
+tutors:
+  - Dominik Brilhaus (CEPLAS)
+organizer:
+  name: Dominik Brilhaus
+  affiliation: CEPLAS Data
+  url: https://www.ceplas.eu/en/research/data-science-and-data-management
+image: ~/assets/images/events/ARCClub.drawio.png
+---
+
+## Setting
+
+- Good internet connection
+- Isolated from lab // office // daily duties
+- Access to the data to be ARCified
+(file share, hard disks)
+- Agile, communicative, collaborative, positive
+- Technical *ad-hoc* support by DataPLANT
+
+## Goals
+
+- Move existing datasets into ARCs
+- Share them via the DataHUB
+- First few steps into ARCs
+- Data users can pick them up from there
+
+## Preparation
+
+- DataHUB user accounts https://register.nfdi4plants.org/
+- DataHUB user group: https://git.nfdi4plants.org/hhu-plant-biochemistry
+- Installation
+  - [ARCitect](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/ARCitect-Manual/index.html)
+  - [Swate](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/SwateManual/Docs01-Installing-Swate.html)
+  - [ARC Commander](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/ArcCommanderManual/index-setup.html)  
+- List of projects to be ARCified
+
+## Why you want to join the **ARC Club**
+
+- You've needed data before, now is your chance to make data available
+- Your contribution to the ARC will be visible
```

Comment:
I think we do not render github style emojies

Reply:
Jup, all old copy / paste from knowledge base

Reply:
I could try adding support with this package: https://github.com/rhysd/remark-emoji